azafea/event_processors/endless/metrics/new-event.rst

```diff
@@ -1,97 +0,0 @@
-========================
-Implementing New Metrics
```

Comment:
I thought it had already been included in the documentation, but you’re right, it’s not. I’ll include it into the Events page.

Reply:
I’ve added a new page in documentation.

Reply:
I **knew** that I had already done this elsewhere, and it’s in `lize/new-protocol-doc`.

I removed the page from this branch, it will be included with the new protocol documentation.